Add Notifi to Notifications Switch Modal

jennifer-notifi opened the pull request:
- Add Notifi to Notifications Switch Modal
- Add Announcements List to the Modal

## **Existing User**
https://www.dropbox.com/s/fpkaif8jrokdiba/FInalVideo.mov?dl=0

## **New User**:
https://www.dropbox.com/s/hl4a7dw0w5d5759/New%20User.mov?dl=0

tradefortendies commented on src/components/NotifiNotifications/Announcements.jsx:
```diff
@@ -0,0 +1,165 @@
+    )
+}
+
+const useStyles = makeStyles((theme) => ({
```
Usually `styles` goes to top of the page.

jennifer-notifi replied:
Thank you for the note! Thought adding it at the end would help with readability, didn't know that should have it at the top! Will update on my end :)

0xM4m4n reviewed (approved):
Looks good to me

tradefortendies commented:
@jennifer-notifi would we still have the announcements page?

thedonmon reviewed (approved):
Looks good

jennifer-notifi commented:
> 

Hello @tradefortendies, we added an announcements modal if users subscribe to Notifi and removed the announcements page. Would you like us to keep the announcements page?

tradefortendies commented:
> > 
> 
> Hello @tradefortendies, we added an announcements modal if users subscribe to Notifi and removed the announcements page. Would you like us to keep the announcements page?

this makes sense
